src/trace_processor/importers/etw/etw_parser.cc

```diff
@@ -38,10 +41,84 @@ namespace perfetto {
 namespace trace_processor {
 
 namespace {
-
 using protozero::ConstBytes;
 
+static constexpr auto kZeroPageCountBlueprint =
+    tracks::CounterBlueprint("zero_page_count",
```

Comment:
I would maybe suggest making it a single "etw_meminfo" blueprint and then adding the counter type as a dimension. I think this is more scalable and would better align with how you would want to query with this.

Reply:
That's much better, thank you.